docs/_articles/en/architecture/architecture.md

```diff
@@ -1,17 +1,17 @@
 ---
-title: Design
+title: Architecture
 lang: en
 ---
 
 ## Salesforce Code Analyzer Architecture
 
-Salesforce Code Analyzer is an open-source [Salesforce CLI plugin](https://developer.salesforce.com/docs/atlas.en-us.sfdx_cli_plugins.meta/sfdx_cli_plugins/cli_plugins_architecture.htm). Most of the code is written in Typescript in the NodeJS framework. Some of the code is written in Java.
+Salesforce Code Analyzer (Code Analyzer) is an open-source [Salesforce CLI plug-in](https://developer.salesforce.com/docs/atlas.en-us.sfdx_cli_plugins.meta/sfdx_cli_plugins/cli_plugins_architecture.htm). Most of the code is written in Typescript in the Node.js framework. Some of the code is written in Java.
 
 ## Salesforce Code Analyzer Internals
 
-The Salesforce Code Analyzer plug-in is powered by multiple static analyzers, also known as rule engines. These rule engines specialize in different aspects of static analysis and support multiple languages. Each rule engine has its own unique set of rules, input parameters, and formats for reporting the results. The Salesforce Code Analyzer plug-in unifies these rules engines as a single static analyzer and provides a common user experience to benefit from the different specialties.
+Code Analyzer is powered by multiple static analyzers, also known as rule engines, and by Salesforce Graph Engine, which provides data flow analysis. These rule engines specialize in different aspects of static analysis and support multiple languages. Each rule engine has its own unique set of rules, input parameters, and formats for reporting the results. Code Analyzer unifies these rules engines as a single static analyzer and provides a common user experience to benefit.
 
-Version {{ site.data.versions.scanner }} of the Salesforce Code Analyzer plug-in uses PMD v{{ site.data.versions.pmd }}, ESlint v{{ site.data.versions.eslint }}, and RetireJS v{{ site.data.versions.retirejs }}.
+Version {{ site.data.versions.scanner }} of Code Analyzer uses PMD v{{ site.data.versions.pmd }}, ESLint v{{ site.data.versions.eslint }}, RetireJS v{{ site.data.versions.retirejs }}, and Salesforce Graph Engine.
```

Comment:
This is the v2 page, and therefore shouldn't mention the graph engine.

Reply:
Got it, editing this out @jfeingold35